src/concurrency/shared_state/arc.md

```diff
@@ -28,8 +28,8 @@ fn main() {
 
 * `Arc` stands for "Atomic Reference Counted", a thread safe version of `Rc` that uses atomic
   operations.
-* `Arc<T>` implements `Clone` whether or not `T` does. It implements `Send` and `Sync` iff `T`
-  implements them both.
+* `Arc<T>` implements `Clone` whether or not `T` does. It implements `Send` and `Sync` iff
+  (if and only if) `T` implements them both.
```

Comment:
Rather than using the short then explaining it, let's just use the long form. I.e. "It implements Send and Sync if and only if T implements them both."

Reply:
Okay, that's simpler indeed.